pilot/pkg/model/sidecar.go

```diff
@@ -217,6 +217,7 @@ func DefaultSidecarScopeForNamespace(ps *PushContext, configNamespace string) *S
 			Hosts: []string{"*/*"},
 		},
 	}
+	// TODO: merge services like sidecar specified using `addService`
```

Comment:
@ramaraochavali @howardjohn WDYT? We are not consistent if a sidecar created with no sidecar

Reply:
yes

Reply:
I thought we did this already... let me dig into it